msi/make.cmd

```diff
@@ -0,0 +1,33 @@
+rmdir /s/q build
+
+rem TODO: Where should the git-credential-winstore, gcm and posh-git binaries
+rem       be found/stored?
+
+rem TODO: Generate GitComponents.wxs from ..\make-file-list.sh
+rem
+rem       1. Files installed to `bin\` folder use Directory='BinFolder' and
+rem          all others use the pattern Directory='INSTALLFOLDER:\path\'.
+rem
+rem       2. The `git.exe` file in `BinFolder` is given the explicit
+rem          Id='GetExe' all other files are anonymous (do not have an Id).
+
+rem Build the .msi.
+rem
+rem TODO: Correctly set the bind paths (change the -b gitfileshere switch)
+rem       to the real source location for the binaries to include in the
+rem       installation package.
```

Comment:
Think of "bind paths" as search paths for binaries to be included into the package. Relative paths in `@Source` and `@SourceFile` attributes the .wxs files are searched from the set of bind paths (provided with the `-b` switch).

So rather than `cd /` in the msi/release.sh, you can add the `/` path as a `-b` switch and that folder will be used to find files. The `getfileshare` value in the `make.cmd` is the one that should be changed.

You can add as many `-b` switches as useful. They are searched in order.